lib/libUPnP/Platinum/Source/Core/PltHttpServer.cpp

```diff
@@ -190,7 +190,7 @@ PLT_HttpServer::ServeFile(const NPT_HttpRequest&        request,
     NPT_FileInfo             file_info;
     
     // prevent hackers from accessing files outside of our root
-    if ((file_path.Find("/..") >= 0) || (file_path.Find("\\..") >= 0)) {
+    if ((file_path.Find("../") >= 0) || (file_path.Find("..\\") >= 0)) {
```

Comment:
this isn't much more then a bandaid, it won't catch any encoded "../" 
see  e.g. https://appcheck-ng.com/url-parsing-path-traversal some examples